docker/docker-compose.lwd.yml

```
@@ -0,0 +1,59 @@
+version: "3.8"
+
+services:
+  zebra:
+    ports:
+      - "8232:8232" # Opens an RPC endpoint (for wallet storing and mining)
```

###### Optional Suggestion

```suggestion
      - "8232:8232" # Opens an RPC endpoint (for lightwalletd and mining)
```